docs/migration/1XX-to-2XX.md

```diff
@@ -24,21 +24,36 @@ for details on how to upgrade to the Firebase `3.x.x` SDK.
 
 ## `$firebaseAuth` Method Renames / Signature Changes
 
+The `$firebaseAuth` service now accepts an optional Firebase `auth` instance instead of a Firebase
```

Comment:
I'd rather use natural language here since `firebase.auth.Auth` doesn't mean much to anyone since the API doesn't look like that. Maybe we can say `firebase.auth()` instance though?